docs/UserGuide.md

```diff
@@ -306,11 +306,29 @@ Examples:
 
 #### 2.2.6 Adding a tag : `tag-add-t`
 
-* (To be filled up by @juayhee)
+Add a tag to a selected task from our task list.
+
+Format: `tag-add-t INDEX [t/TAG]`
+
+Examples:
+* `tag-add-t 1 important` adds the tag "important" to the first task in the list.
+
 
 #### 2.2.7 Deleting a tag : `tag-del-t`
 
-* (To be filled up by @juayhee)
+Delete a tag from a selected task in our task list.
+
+Format: `tag-del-t INDEX [t/TAG]`
```

Comment:
Should this be tag-del-t INDEX TAG_NAME instead?

Reply:
Fixed